src/libs/actions/Chronos.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'underscore';
+import CONST from '../../CONST';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+
+/**
+ @param {String} reportID
+ @param {String} eventID
+ * @param {Object} reportAction
+ * @param {String} reportAction.sequenceNumber
+ * @param {Object[]} events
+ */
+const removeEvent = (reportID, eventID, reportAction, events) => {
```

Comment:
`reportAction` is not required here, use `reportActionID` instead.

Also, it may be better to change the order of params to: `reportID, reportActionID, eventID, events`

Reply:
You're right, good idea.